addons/radar/XEH_preInit.sqf

```diff
@@ -197,6 +215,15 @@ private _curCat = localize "STR_dui_cat_compass";
     ,false
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(icon_scale_crew)
+    ,"SLIDER"
+    ,[localize "STR_dui_compass_icon_scale_crew", localize "STR_dui_compass_icon_scale_crew_desc"]
+    ,[CBA_SETTINGS_CAT, _curCat]
+    ,[0.01, 24, 6, 2]
```

Comment:
After checking it out, 6 is quite large, at least on 1080p with small UI scaling.

Reply:
OH, I see. It does scale with the crew range. It is find on 6 and 500 meters range, my apologies.